appcircle_ios_appdome/1.0.1/component.yaml

```diff
@@ -0,0 +1,88 @@
+platform: iOS
+buildPlatform:
+displayName: Appdome Build-2Secure for iOS
+description: "Integration that allows activating security and app protection features, building and signing mobile apps using Appdome's API."
+docsUrl: https://docs.appcircle.io/workflows/ios-specific-workflow-steps#app-center-ios-distribution
```

Comment:
@boztopuz This doc should be change with this:
- https://docs.appcircle.io/workflows/ios-specific-workflow-steps#appdome-build-2secure-for-ios